larray/viewer/view.py

```diff
@@ -1276,6 +1280,26 @@ def void_formatter(array, *args, **kwargs):
 
         # Make the dialog act as a window
         self.setWindowFlags(Qt.Window)
+
+        # load file if any
+        if isinstance(data, str):
+            if os.path.isfile(data):
+                self._open_file(data)
+            else:
+                QMessageBox.critical(self, "Error", "File {} could not be found".format(data))
+                self.new()
+        # load last opened session if any
+        elif data is REOPEN_LAST_FILE:
```

Comment:
In that case, if the list of recent files is empty, we will have the error message "Error - File ... could not be found".